src/toastfile.rs

```diff
@@ -404,11 +471,13 @@ fn check_task(name: &str, task: &Task) -> Result<(), Failure> {
     // Check `mount_paths`.
     for path in &task.mount_paths {
         // Check that the path doesn't contain any commas. [tag:mount_paths_no_commas]
-        if path.to_string_lossy().contains(',') {
+        if path.container_path.to_string_lossy().contains(',')
+            || path.host_path.to_string_lossy().contains(',')
+        {
             return Err(Failure::User(
                 format!(
-                    "Mount path {} of task {} has a {}.",
-                    path.to_string_lossy().code_str(),
+                    "Mount path {:?} of task {} has a {}.",
```

Comment:
I missed this: we shouldn't rely on `std::fmt::Debug` in production. (I fixed it in https://github.com/stepchowfun/toast/pull/365.)